test.js

```diff
@@ -160,8 +160,8 @@ describe('Tests mongoose-timezone-offset', function() {
     });
 
     describe('edge cases', function() {
-        it('Should not break with bad offset of 17', async function() {
```

Comment:
ffgff

---
**Sema Reaction:** :ok_hand: Looks good | **Sema Tags:** Secure, Inefficient, Inelegant, Not reusable, Fault-tolerant, Maintainable